src/context/search-context.js

```diff
@@ -118,7 +118,7 @@ export async function querySearch(state, dispatch) {
   }
 }
 
-export function getSnippetResultOrExcerpt(hitType, hit) {
+export function getHighlightResultOrExcerpt(hitType, hit) {
```

Comment:
@jasonlcrane I think you were missing my last commit on this change, this may break the search results page.